src/utils/ddb/tests/ddb_commands_tests.c

```diff
@@ -389,8 +389,11 @@ dcv_suit_teardown(void **state)
 {
 	struct dt_vos_pool_ctx *tctx = *state;
 
-	if (tctx == NULL)
+	if (tctx == NULL) {
 		fail_msg("Test not setup correctly");
+		return -DER_UNKNOWN;
```

Comment:
I understand this is just test code, but DER_UNKNOWN is not friendly for debugging or triaging purposes. please consider changing that in a follow on